packages/beacon-node/src/api/rest/base.ts

```diff
@@ -106,8 +99,7 @@ export class RestApiServer {
     server.addHook("onError", async (req, _res, err) => {
       // Don't log ErrorAborted errors, they happen on node shutdown and are not useful
       // Don't log NodeISSyncing errors, they happen very frequently while syncing and the validator polls duties
-      // Don't log eventstream aborted errors if server instance is being closed on node shutdown
-      if (err instanceof ErrorAborted || err instanceof NodeIsSyncing || this.status === Status.Closed) return;
```

Comment:
I added this in https://github.com/ChainSafe/lodestar/pull/5330 to avoid noisy errors on shutdown, however eventstream aborts are now properly handled and no more error are logged on shutdown.

Since we attempt to gracefully close connections now, it is highly unlikely to log any errors on shutdown since we should only get `ErrorAborted` which is already checked and not logged. Any error that is not caught here is unexpected and should be logged.